Dockerfile

```diff
@@ -27,20 +27,19 @@ RUN apt-get install -y mafft && \
     mafft --version
     
 RUN apt-get install -y fasttree && \
-    ln -s /usr/bin/fasttree /usr/bin/FastTree && \
     FastTree -expert
 
 RUN mkdir -p /kb/deps
 WORKDIR /kb/deps
 
 RUN wget --quiet https://github.com/davidemms/OrthoFinder/releases/download/2.5.2/OrthoFinder_source.tar.gz && \
     tar -zxf OrthoFinder_source.tar.gz && \
-    mv OrthoFinder_source /kb/deployment/bin/orthofinder
+    mv OrthoFinder_source /usr/bin/orthofinder
```

Comment:
I realise this is existing code, but anyway -- it's better to try to concatenate the RUN statements as much as possible to reduce the number of layers in the resultant image (each RUN produces a new layer).

Reply:
Isn't that what's happening with the use of `&& \` ?

Reply:
No -- I mean concatenating the RUN statements themselves, e.g.

```
RUN apt-get install -y fasttree && \
     FastTree -expert

 RUN mkdir -p /kb/deps
 WORKDIR /kb/deps

 RUN wget --quiet https://github.com/davidemms/OrthoFinder/releases/download/2.5.2/OrthoFinder_source.tar.gz && \
```
==>
```
RUN apt-get install -y fasttree && \
    FastTree -expert && \
    mkdir -p /kb/deps && \
    cd /kb/deps && \
    wget --quiet https://github.com/davidemms/OrthoFinder/releases/download/2.5.2/OrthoFinder_source.tar.gz && \
```

Reply:
If there are a large number of RUN statements it's worth considering whether there should be a set up script, especially if developers might also install the software locally during development and testing.

Reply:
This will improve performance and make the docker image smaller ==> please implement.

Reply:
Updated to run in a single script and tested in GitHub Actions (9ca841d60089414785b076375ae088febb9d0668)